Combat/Include/Spell.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include "Card.hpp"
+#include <iostream>
+
+class Spell : public Card
+{
+   public:
+      uint power;
+      virtual void useCard() = 0;
+};
+//TODO write subclasses (damageSpell, healingSpell ...)

```

Comment:
Jeżeli tak to co będzie częścią wspólną tych spelli? Bo jeśli powet to w takim razie co z kartami, które np zmuszają do odrzucenia karty, pozwalają dobrać kartę z ręki przeciwnika itd?